### PR TITLE
Feature/program

### PR DESCRIPTION
**Pull request by mnavyasree1906**
This PR includes the implementation of Orange HRM login function.

### CODE REVIEW

File: 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	public static void main(String[] args) {
+		
+		//1.Launch Chrome Browser
+		WebDriver driver=new ChromeDriver();
+		
+		//maximize the browser(Method chaining)
+		driver.manage().window().maximize();
+		
+		//2. Navigate to OrangeHRM url
+		driver.get("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
```
Comment (linuxramu) [inline 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java:21]: dont hardcode it.. move this into properties file !   the way is : create a prop file and import that into this java file

File: 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		
+		//wait in Login Page
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(3));
```
Comment (linuxramu) [inline 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java:24]: move this as variable into prop file again...... for ex : 3 is hardcoded !

File: 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        password.sendKeys("admin123");
+        
+        //7. Locate login button
+        WebElement loginButton = driver.findElement(By.xpath("//button[@type='submit']"));
+        
+       //7. click the button
+        loginButton.click();
+		
+	// 8. Assert for success message
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
```
Comment (linuxramu) [inline 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java:45]: we will have to use **assertion** for this ! ... you need to run it as Test Project !   examples : Assert, AssterFail.. something like that !  watch TestNG or similar framework videos !

**Review by linuxramu — CHANGES_REQUESTED** [review]
Address all the changes
1. Prop file
2. TestNG and add assertion

**Comment by mnavyasree1906** [comment]
> Address all the changes
> 
> 1. Prop file
> 2. TestNG and add assertion

I made all the changes in 002-loginwithtestng

File: 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java (hunk 3 above)
Comment (mnavyasree1906) [reply 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java:45]: I made all the changes in 002-loginwithtestng

File: 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java (hunk 2 above)
Comment (mnavyasree1906) [reply 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java:24]: I made all the changes in 002-loginwithtestng

File: 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java (hunk 1 above)
Comment (mnavyasree1906) [reply 001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java:21]: I made all the changes in 002-loginwithtestng

**Comment by mnavyasree1906** [comment]
As per the PR 1 review comments, I have implemented all the changes in the '002-loginwithtestng'.

**Pull request merged**